### ISSUE TITLE
Warning "A restricted method in java.lang.System has been called" is printed with Java24

### ISSUE THREAD

**Issue by paul-dingemans** [pinterest/ktlint#3041]
When invoking `ktlint --help` on Java 24, the warning "A restricted method in java.lang.System has been called" is printed:
```
WARNING: A restricted method in java.lang.System has been called
WARNING: java.lang.System::load has been called by com.sun.jna.Native in an unnamed module (file:/Users/pdingemans/git/ktlint/ktlint/ktlint-cli/build/run/ktlint)
WARNING: Use --enable-native-access=ALL-UNNAMED to avoid a warning for callers in this module
WARNING: Restricted methods will be blocked in a future release unless native access is enabled

Usage: ktlint [<options>] [<arguments>]... <command> [<args>]...

  An anti-bikeshedding Kotlin linter with built-in formatter. (https://pinterest.github.io/ktlint/latest/).

...
```

This warning is caused by an [issue in clikt](https://github.com/ajalt/clikt/issues/604).
